lib/bus.js

```diff
@@ -683,18 +683,27 @@ module.exports = function bus(conn, opts) {
                             let propAccessMode = Object.keys(propObj)[0]
                             let propType = propObj[propAccessMode]
 
+                            //get user-defined getter/setters
+                            let propGetter = obj[ifaceName].__lookupGetter__(propName)
+                            let propSetter = obj[ifaceName].__lookupSetter__(propName)
+
+                            //use default getter/setter if not user-defined
+                            if (!propGetter) {
```

Comment:
In this particular case I can be more explicit, but in lots of cases, I prefer to coerse to boolean since the input value could be ```undefined``` as well as ```null```. If the input value is to be a number, then the more specific ```isNaN``` should be used.